src/renderers/shared/stack/reconciler/__tests__/ReactErrorBoundaries-test.js

```diff
@@ -84,12 +103,15 @@ describe('ReactErrorBoundaries', function() {
       }
     }
 
-    var EventPluginHub = require('EventPluginHub');
     var container = document.createElement('div');
-    EventPluginHub.putListener = jest.fn();
     container.innerHTML = ReactDOMServer.renderToString(<Boundary />);
     expect(container.firstChild.innerHTML).toBe('Happy Birthday!');
-    expect(EventPluginHub.putListener).not.toBeCalled();
+    // note: renderToString does not call componentDidMount
```

Comment:
OK, updated the test to use `ReactDOM.render` instead. FYI, original test was introduced by https://github.com/facebook/react/pull/6694